egs/babel_multilang/s5/local/prepare_lang_conf.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+# This script maps lang-name to its config w.r.t fullLP or limitedLP condition.
+
+fullLP=true
+. ./utils/parse_options.sh
```

Comment:
I'm puzzled by this --  why is not asking user to provide lang.conf in data/$lang/ not good enough? Especially when user has to set up the directories manually anyway? 
People who are using the babel scripts have their own configs (named differently), so I think enforcing the configs named in a certain pattern will be much higher burden for them. Also, it would allow you to drop the FullLP and LimitedLP distinction + allowed to add different datasets into the mix. BBN for example adds Fisher for the BNF training.